frontend/src/components/AboutUs/AboutUs.tsx

```diff
@@ -17,10 +18,40 @@ const AboutUs: React.FC = () => (
             situações vulneráveis. Nossa missão é proporcionar um lar seguro e
             amoroso para cada ser que cruzar nosso caminho.
           </p>
+          <div className="about-us__container__body__button">
+            <Button className="about-us__container__body__button__about">
+              <a href="#" target="_blank">
+                Conheça nosso instagram
+              </a>
+            </Button>
+            <Button className="about-us__container__body__button__contact">
+              <a href="#" target="_blank">
+                Contate-nos no email
+              </a>
+            </Button>
+          </div>
         </div>
-        <div className="about-us__container__body__rigth">
+        <div className="about-us__container__body__right">
           <h2>Nossa missão</h2>
           <p>Além do resgate e do incentivo a adoção, nós também:</p>
+          <div className="about-us__container__body__button">
+            <Button className="about-us__container__body__button-card">
```

Comment:
Por que botões com plain text no meio ao envés de uma `<div>` com um `<p>`?

Acredito que otimizar o HTML para ser mais semântico é complexo o suficiente para ser propositalmente adiado até um refactor, mas certas coisas são desnecessariamente complicadas.

Note que usar esse `<Button>` não é diferente de usar um `<p>`, e que o fato destes cards não possuírem funcionalidade vai forçar um refactor futuro, já que `<Button>` tem a responsabilidade de agir como um botão, não como um container genérico para texto; e que **botões** podem possuir funcionalidades genéricas apropriadas para botões, coisa que não é razoável de se remover ou desativar por conta de um caso de uso onde um botão não se comporta como um botão.

Se tiver de usar um botão cru com algumas classes use o `button` padrão do HTML. Se tiver de criar um botão com funcionalidade especial, considere contribuir para o botão genérico primeiro.

Reply:
Fala Marcos.
Poderia ser feito sim, no caso apenas reutilizei um componente já existente, mas sim, faz sentido o que você disse!

Ao invés de eu criar um `<p>`, melhor um componente Card para ser reutilizado.

Apenas quis fazer o uso do que o React proporciona, componentes, só reutilizei um existente, mas realmente, a ideia melhor seria criar outro componente para reutilizar.


Obrigado pelo feedback!

Reply:
Concordo.

> Ao invés de eu criar um `<p>`, melhor um componente Card para ser reutilizado.

Estava falando da estrutura final do HTML, não do React. Sua ideia faz sentido, independentemente.

Concordo que um `Card` pode ser útil no futuro e é bom estabelecer o uso de componentes comuns.

A nomenclatura "Card" é perfeita.

Reply:
Top!

Já criei e já subi o push! Depois dou uma olhada melhor sobre o que comentou do css